crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3458,11 +3409,10 @@ impl AuthorityPerEpochStore {
             self.record_end_of_message_quorum_time_metric();
         }
 
-        Ok([
-            verified_non_randomness_transactions,
-            verified_randomness_transactions,
-        ]
-        .concat())
+        let mut all_txns = verified_non_randomness_transactions;
+        all_txns.extend(verified_randomness_transactions);
```

Comment:
```suggestion
        let all_txns = [
            verified_non_randomness_transactions,
            verified_randomness_transactions,
        ]
        .concat();
```

does this work and avoid the `mut`?